atomic_reactor/plugins/koji_parent.py

```diff
@@ -23,7 +23,7 @@
 import time
 
 
-DEFAULT_POLL_TIMEOUT = 60 * 10  # 10 minutes
+DEFAULT_POLL_TIMEOUT = 60 * 3  # 3 minutes
```

Comment:
Why this change?

Reply:
it was case for build which didn't have brew build in koji

not sure why it was even 10m  I think 3m is just fine

Reply:
unrelated change should be in separate commit

Reply:
well it is sort of related

Reply:
how?

Reply:
because konflux images don't have brew build, so this is just making it wait longer for non-existing koji build

Reply:
ah so if iamge is not in koji, then we will wait everytime 3 minutes?

Reply:
exactly, and 10m seemed quite a lot so I've lowered it

Reply:
I'm thinking if we even need this timeout, isn't  an leftover from autorebuilds feature which has been dropped?

Reply:
I'd leave it, 3m is fine

Reply:
it may happen that image used by build is still building, so we can give it 3m to finish

Reply:
okay, 3min sounds reasonable to finish build after image is pushed to registry in build process